lib/daemon.py

```diff
@@ -4,48 +4,22 @@
 
 from systemd.daemon import notify  # type: ignore
 
-from uaclient import defaults, http
 from uaclient.config import UAConfig
-from uaclient.daemon import (
-    poll_for_pro_license,
-    retry_auto_attach,
-    setup_logging,
-)
+from uaclient.daemon import poll_for_pro_license, retry_auto_attach
 
-LOG = logging.getLogger("ubuntupro.lib.daemon")
+LOG = logging.getLogger("ubuntupro.daemon")
+root_logger = logging.getLogger("ubuntupro")
 
 
 def main() -> int:
-    setup_logging(
-        logging.INFO,
-        logging.DEBUG,
-        defaults.CONFIG_DEFAULTS["daemon_log_file"],
-        logger=LOG,
-    )
+    LOG.setLevel(logging.DEBUG)
```

Comment:
We still need our formatter and redaction filter - I think we still need a daemon.setup_logging or similar function